### ISSUE TITLE
[Feature Request]: implement new method for factor analysis using out of sample prediction errors

### ISSUE THREAD

**Issue by TarandeepKang** [jasp-stats/jasp-issues#2261]
### Description

New method for factor analysis

### Purpose

Authors report that this method slightly outperforms others for exploratory factor analysis including parallel analysis and exploratory graph analysis

### Use-case

In similar circumstances to typical factor analysis

### Is your feature request related to a JASP module?

Factor

### Describe the solution you would like

Implement this method as described in the paper below and the package the authors have developed

### Additional context

Haslbeck, J. M. B., & Van Bork, R. (2022). Estimating the number of factors in exploratory factor analysis via out-of-sample prediction errors. Psychological Methods. https://doi.org/10.1037/met0000528

**Comment by juliuspfadt** [comment]
Good idea. Although I wonder if we should wait for replications maybe.

**Comment by TarandeepKang** [comment]
Will be a reasonable idea to wait for some replications, although you are probably in a better position to know when they will come, because isn't the first author of the paper at Amsterdam?
